lib/basedriver/driver.js

```diff
@@ -20,6 +21,7 @@ B.config({
   cancellation: true,
 });
 
+const BASEDRIVER_VER = require(path.resolve(__dirname, '..', '..', '..', 'package.json')).version;
```

Comment:
chromedrivers uses a bit different approach: https://github.com/appium/appium-chromedriver/blob/78b0ed6ae4d973783733eb626ae14b5b2edc716c/lib/utils.js#L9

which one is right?

Reply:
right now i'm assuming that everything is run transpiled. if we're ever in a scenario where that's not the case, the test will tell us.

Reply:
I think chromedriver cares because the install script needs to run untranspiled